codeformat/dokka/styles/prism.css

```diff
@@ -0,0 +1,585 @@
+/* PrismJS 1.14.0
+http://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript+abap+actionscript+ada+apacheconf+apl+applescript+c+arff+asciidoc+asm6502+aspnet+autohotkey+autoit+bash+basic+batch+bison+brainfuck+bro+cpp+csharp+arduino+coffeescript+clojure+ruby+csp+css-extras+d+dart+diff+django+docker+eiffel+elixir+elm+markup-templating+erlang+fsharp+flow+fortran+gedcom+gherkin+git+glsl+go+graphql+groovy+haml+handlebars+haskell+haxe+http+hpkp+hsts+ichigojam+icon+inform7+ini+io+j+java+jolie+json+julia+keyman+kotlin+latex+less+liquid+lisp+livescript+lolcode+lua+makefile+markdown+erb+matlab+mel+mizar+monkey+n4js+nasm+nginx+nim+nix+nsis+objectivec+ocaml+opencl+oz+parigp+parser+pascal+perl+php+php-extras+sql+powershell+processing+prolog+properties+protobuf+pug+puppet+pure+python+q+qore+r+jsx+typescript+renpy+reason+rest+rip+roboconf+crystal+rust+sas+sass+scss+scala+scheme+smalltalk+smarty+plsql+soy+stylus+swift+tcl+textile+twig+tsx+vbnet+velocity+verilog+vhdl+vim+visual-basic+wasm+wiki+xeora+xojo+yaml&plugins=line-numbers+toolbar+show-language */
+/**
+ * prism.js default theme for JavaScript, CSS and HTML
+ * Based on dabblet (http://dabblet.com)
+ * @author Lea Verou
+ */
+
+/*
+* Dracula Theme for Prism.JS
+*
+* @author Gustavo Costa
+* e-mail: gusbemacbe@gmail.com
+*
+* @contributor Jon Leopard
+* e-mail: jonlprd@gmail.com
+*
+* @license MIT 2016-2020
+*/
+
+/* Scrollbars */
+
+:root
+{
+	--background: #282A36;
+	--comment:    #6272A4;
+	--foreground: #F8F8F2;
+	--selection:  #44475A;
+
+	--cyan:       #8BE9FD;
+	--green:      #50FA7B;
+	--orange:     #FFB86C;
+	--pink:       #FF79C6;
+	--purple:     #BD93F9;
+	--red:        #FF5555;
+	--yellow:     #F1FA8C;
+
+	/* Transparency */
+
+	/** 30% of transparency **/
+	--background-30: #282A3633;
+	--comment-30:    #6272A433;
+	--foreground-30: #F8F8F233;
+	--selection-30:  #44475A33;
+
+	--cyan-30:       #8BE9FD33;
+	--green-30:      #50FA7B33;
+	--orange-30:     #FFB86C33;
+	--pink-30:       #FF79C633;
+	--purple-30:     #BD93F933;
+	--red-30:        #FF555533;
+	--yellow-30:     #F1FA8C33;
+
+	/** 40% of transparency **/
+	--background-40: #282A3666;
+	--comment-40:    #6272A466;
+	--foreground-40: #F8F8F266;
+	--selection-40:  #44475A66;
+
+	--cyan-40:       #8BE9FD66;
+	--green-40:      #50FA7B66;
+	--orange-40:     #FFB86C66;
+	--pink-40:       #FF79C666;
+	--purple-40:     #BD93F966;
+	--red-40:        #FF555566;
+	--yellow-40:     #F1FA8C66;
+}
+
+pre::-webkit-scrollbar
+{
```

Comment:
Is this bracket placement the standard for css? Else I think it should mirror what we do in the rest of the codebase

Reply:
The Prism theme is taken from [dracula](https://draculatheme.com/prism) with the file itself being taken from [unpkg](https://unpkg.com/dracula-prism@2.1.13/dist/css/dracula-prism.css) in order to work with Dokka. I left it unmodified, but due to how Dokka is set up, I can't simply apply the stylesheet link the way it's intended.

Reply:
o7 all good